SweetPea to English/gpt3prompt_derived_factors.txt

```diff
@@ -282,3 +284,118 @@ Text:
 The color response factor depends on the color factor and has two levels: -1 and 1. If the color is red or blue, then the corresponding color response is -1 or 1, respectively.
 
 Code:
+col      = factor("Color",  [Level("red",2), "green", "blue", "brown"])
+wrd       = factor("Word", ["red", "green", "blue", "brown"])
+
+def is_response_left(col_factor):
+    return col_factor == "red"
+def is_response_right(col_factor):
+    return col_factor == "green"
+def is_response_up(col_factor):
+    return col_factor == "blue"
+def is_response_down(col_factor):
+    return col_factor == "brown"
+
+resp = factor("response", [
+    derived_level("left", within_trial(is_response_left, [col]),2),
+    derived_level("right", within_trial(is_response_right, [col])),
+    derived_level("up", within_trial(is_response_up, [col])),
+    derived_level("down", within_trial(is_response_down, [col]))
+])
+
+Text:
+The response factor depends on the Color factor and has four levels: left, right, up, and down. If the Color is red, green, blue, or brown, then the corresponding response is left, right, up, and down, respectively. The response left occurs twice as often as the response right, up or down.
+
+Code:
+color      = factor("Color",  ["red", "green", "blue", "brown"])
+word       = factor("Word", ["red", "green", "blue", "brown"])
+
+def is_congruent(color, word):
+    return color == word
+
+def is_incongruent(color, word):
+    return not is_congruent(color, word)
+
+congr_level = derived_level("congruent", within_trial(is_congruent, [color, word]),weight=4)
+incongr_level = derived_level("incongruent", within_trial(is_incongruent, [color, word]))
+
+congr_factor = factor("congruency", [
+    congr_level,
+    incongr_level
+])
+
+Text:
+The congruency factor depends on the regular factors color and word and has two levels: congruent and incongruent. A trial is considered congruent if the color matches the word, otherwise, it is considered incongruent. In our experiment the frequency between congruent and incongruent trials is different. There are four times as many congruent trials then incongruent trials.
```

Comment:
```suggestion
The congruency factor depends on the regular factors color and word and has two levels: congruent and incongruent. A trial is considered congruent if the color matches the word, otherwise, it is considered incongruent. The trial sequence included four times as many congruent trials as incongruent trials.
```